data/templates/template.typst

```diff
@@ -16,36 +16,62 @@
   keywords: (),
   date: none,
   abstract: none,
+  thanks: none,
   cols: 1,
   margin: (x: 1.25in, y: 1.25in),
   paper: "us-letter",
   lang: "en",
   region: "US",
   font: (),
   fontsize: 11pt,
+  mathfont: "New Computer Modern Math",
+  codefont: "DejaVu Sans Mono",
+  linestretch: 1,
   sectionnumbering: none,
+  linkcolor: ["#800000"],
+  citecolor: ["#0000FF"],
+  toccolor: ["#800000"],
   doc,
 ) = {
   set document(
     title: title,
-    author: authors.map(author => content-to-string(author.name)),
+    author: authors.map(author => content-to-string(author.name)).join(", ", last: ", and "),
```

Comment:
I don't like this because `and` is an English word; anyone writing in another language would need to change the template.  We could consider using `&`, or just use commas.